scripts/filterToRule.js

```diff
@@ -0,0 +1,246 @@
+/*
+ * Convert adblock filters to regex rules
+ * node filterToRule.js -f <filterlist> -o <outputfile> -c <path to tracker file> -t <rule type: 'rule' or 'whitelist'>
+ */
+
+const program = require('commander')
+const fs = require('fs')
+const merge = require('deepmerge')
+const _ = require('underscore')
+const assert = require('assert')
+const utils = require('../shared/js/background/utils.es6')
+const RandExp = require('randexp')
```

Comment:
After `randexp` was plugged in `filterToRule` script started failing randomly because some of the generated URLs are invalid. Examples:

- http://coremetrics.com+cO|6;!B-'LM(z(u7Zq=1=piDNa=r_(=J$/eluminate.js
- http://ad4game.comZ1N*@+S/vS|LnnK~f/:W?=Z)fo&|E]h>s/images/

Both of those work when we add a `/` after `.com`. Also both of those end up in `unMatchedRules.json`. This might be suggesting some underlying issue with the generated regexes (or the input data?).

Reply:
Nice catch. Looks like the input filters are bad. ` ||ad4game.com*/images/` `||coremetrics.com*/eluminate.js`. I'll fix them and rerun the script.